### PR TITLE
My Portfolio Second version

### PR DESCRIPTION
**Pull request by Agape-nduwamungu**
Ensured:
- that the added part for index and styles files is visible.
- that the linters are working correctly.

### CODE REVIEW

**Comment by Agape-nduwamungu** [comment]
Had some conflicts.

File: index.html [hunk 1]
```diff
@@ -55,7 +55,151 @@ <h1>Hey there. I'm Agape. <br><span class="heading2">I'm a software developer</s
...
+                            <li class="list1">Ruby on Rails</li>
+                            <li class="list1">CSS</li>
+                            <li class="list1">Javascript</li>
+                            <li class="list1">HTML</li>
+                        </ul>
+                        <button class="project-card-btn button popup" type="button">See Project</button>
+                    </span>
+                </div>
+                <div class="project-card">
+                    <img src="images/placeholder.svg" alt="placeholder1" class="project-2-img" />
```
Comment (zieeco) [inline index.html]: - Hey, the images you've added here do not exits in your image folder. please, kindly check to add them as they are broken right now.

File: index.html [hunk 2]
```diff
@@ -55,7 +55,151 @@ <h1>Hey there. I'm Agape. <br><span class="heading2">I'm a software developer</s
...
+                            <li class="list2">Ruby on Rails</li>
+                            <li class="list2">CSS</li>
+                            <li class="list2">Javascript</li>
+                            <li class="list2">HTML</li>
+                        </ul>
+                        <button class="project-card-btn button popup" type="button">See Project</button>
+                    </span>
+                </div>
+                <div class="project-card">
+                    <img src="images/placeholder.svg" alt="placeholder1" class="project-3-img" />
```
Comment (zieeco) [inline index.html]: - Hey, the images you've added here do not exits in your image folder. please, kindly check to add them as they are broken right now.

File: index.html [hunk 3]
```diff
@@ -55,7 +55,151 @@ <h1>Hey there. I'm Agape. <br><span class="heading2">I'm a software developer</s
...
+                            <li class="list3">Ruby on Rails</li>
+                            <li class="list3">CSS</li>
+                            <li class="list3">Javascript</li>
+                            <li class="list3">HTML</li>
+                        </ul>
+                        <button class="project-card-btn button popup" type="button">See Project</button>
+                    </span>
+                </div>
+                <div class="project-card">
+                    <img src="images/placeholder.svg" alt="placeholder1" class="project-4-img" />
```
Comment (zieeco) [inline index.html]: - Hey, the images you've added here do not exits in your image folder. please, kindly check to add them as they are broken right now.

File: index.html [hunk 4]
```diff
@@ -55,7 +55,151 @@ <h1>Hey there. I'm Agape. <br><span class="heading2">I'm a software developer</s
...
+                            <li class="list4">Ruby on Rails</li>
+                            <li class="list4">CSS</li>
+                            <li class="list4">Javascript</li>
+                            <li class="list4">HTML</li>
+                        </ul>
+                        <button class="project-card-btn button popup" type="button">See Project</button>
+                    </span>
+                </div>
+                <div class="project-card">
+                    <img src="images/placeholder.svg" alt="placeholder1" class="project-5-img" />
```
Comment (zieeco) [inline index.html]: - Hey, the images you've added here do not exits in your image folder. please, kindly check to add them as they are broken right now.

File: index.html [hunk 5]
```diff
@@ -55,7 +55,151 @@ <h1>Hey there. I'm Agape. <br><span class="heading2">I'm a software developer</s
...
+                            <li class="list5">Ruby on Rails</li>
+                            <li class="list5">CSS</li>
+                            <li class="list5">Javascript</li>
+                            <li class="list5">HTML</li>
+                        </ul>
+                        <button class="project-card-btn button popup" type="button">See Project</button>
+                    </span>
+                </div>
+                <div class="project-card">
+                    <img src="images/placeholder.svg" alt="placeholder1" class="project-6-img" />
```
Comment (zieeco) [inline index.html]: - Hey, the images you've added here do not exits in your image folder. please, kindly check to add them as they are broken right now.

File: index.html [hunk 6]
```diff
@@ -55,7 +55,151 @@ <h1>Hey there. I'm Agape. <br><span class="heading2">I'm a software developer</s
...
+                    <span class="proj6">
+                        <h1 class="project-card-title">Multi-post Stories <br> Gain+Glory</h1>
+                        <ul class="project-card-list">
+                            <li class="list6">Ruby on Rails</li>
+                            <li class="list6">CSS</li>
+                            <li class="list6">Javascript</li>
+                            <li class="list6">HTML</li>
+                        </ul>
+                        <button class="project-card-btn button popup" type="button">See Project</button>
```
Comment (zieeco) [inline index.html]: ### ALL LIST ITEMS AND BUTTONS IN THE WORKS SECTION
- 
- Hey, please, let's style the list of items and align the buttons correctly to match the design as much as we can to improve our design. kindly check the images below:

*your design*
Screenshot from 2023-03-25 22-52-57

*Figma design*
Screenshot from 2023-03-25 22-52-35

File: index.html [hunk 7]
```diff
@@ -55,7 +55,151 @@ <h1>Hey there. I'm Agape. <br><span class="heading2">I'm a software developer</s
...
+                            <li class="about-list">Data Management</li>
+                            <li class="about-list">Version Control</li>
+                            <li class="about-list">Web Development</li>
+                            <li class="about-list">CLI</li>
+                            <li class="about-list">API Design</li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </section>
```
Comment (zieeco) [inline index.html]: - Hey, kindly improve your about section. it's missing all the tags on the cards and the images are not showing including the background image. please, let's work on this to improve our design. chek the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1-468&t=zM3zI9tBvX4156tQ-0) to improve your design.

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Agape-nduwamungu ,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. ✔️
- Descriptive PR. ✔️
- Good commit message :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions
- N/A

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Agape-nduwamungu** [comment]
Opened a new pull request

**Pull request closed without merging**